src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -0,0 +1,86 @@
+const BinarySocket     = require('../../../../base/socket');
+const localize         = require('../../../../../_common/localize').localize;
+const Url              = require('../../../../../_common/url');
+
+const AccountClosure = (() => {
+    const form_selector = '#form_closure';
+
+    const onLoad = () => {
+        $(form_selector).on('submit', (event) => {
+            event.preventDefault();
+            submitForm();
+        });
+        $('#closure-container').setVisibility(1);
+        $('#other-reason').on('keyup', () => $('#other-reason').removeClass('error-field'));
```

Comment:
better store `$('#other-reason')` in a variable so we don't have to retrieve it every time on keyup

Reply:
Yeah, should be better. I tried defining it globally, but didn't work. I'll declare it and then assign its value on load

Reply:
that's because jquery hasn't loaded in the global yet, so we can't assign jquery globally. you can create a let variable globally and set the jquery value onload as well.

Reply:
Yeah, I thought so too